src/nexmo/__init__.py

```diff
@@ -102,7 +98,7 @@ def __init__(
 
         self.api_host = "api.nexmo.com"
 
-        user_agent = "nexmo-python/{version} python/{python_version}".format(
+        user_agent = "nexmo-python/{version}/{python_version}".format(
```

Comment:
Please don't change the User Agent. Every SDK and sample app sets this and we use it for reporting. See https://github.com/Nexmo/repo-standards/blob/master/set-user-agent.md for more information and examples.

Reply:
Done!